src/FunctionApps/PITest_FunctionApp/tests/test_decrypt_function.py

```diff
@@ -77,3 +78,74 @@ def test_decrypt_message_failure_wrong_receiver(local_settings):
         )
 
     assert "Cannot decrypt the provided message with this key" in str(exc_info.value)
+
+
+def test_trigger_success(local_settings):
+    """Test function trigger conditions
+
+    Args:
+        local_settings ([type]): passed automatically via above fixture
+    """
+    test_file_path = (
+        files("DecryptFunction").parent / "tests" / "assets" / "encrypted.txt"
```

Comment:
Do you need to change this to `Path("DecryptFunction").parent ...` like you did on line 27?

Reply:
Good callout - thanks! Graham's recent change also used this [here](https://github.com/CDCgov/prime-public-health-data-infrastructure/pull/42/files). I do think we'll want to fix this more holistically in the future / using package structure, but if this works for now in VSCode, CLI, and when running we should use it instead